tests/unit/commando/core_test.py

```diff
@@ -1,6 +1,23 @@
-from commando.core import hello_world
+from unittest.mock import sentinel
 
+import pytest
 
-class TestHelloWorld:
+from commando.core import commando
+
+
+class TestCommando:
     def test_it(self):
-        assert hello_world() == "Hello, world!"
+        exit_status = commando(
+            [sentinel.repo_1, sentinel.repo_2],
+            ["test", "command"],
+            sentinel.commit_message,
+            sentinel.branch,
+            sentinel.pr_title,
+            sentinel.pr_body,
+        )
+
+        assert not exit_status
+
+    @pytest.fixture(autouse=True)
+    def subprocess(self, mocker):
+        return mocker.patch("commando.core.subprocess")
```

Comment:
There are not really proper tests of either the CLI or the core code. I'm not sure it's really worth the bother for this. Perhaps we can get it working and see if it's useful and maybe send proper tests in a follow-up PR